src/main/java/net/io/kino/model/EmailConfirmation.java

```diff
@@ -0,0 +1,55 @@
+package net.io.kino.model;
+
+import net.io.kino.KinoApplication;
+import org.springframework.boot.SpringApplication;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.mail.*;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeBodyPart;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.MimeMultipart;
+import java.io.IOException;
+import java.util.Date;
+import java.util.Properties;
+
+public class EmailConfirmation {
+
+    static public void sendmail() throws MessagingException, IOException {
+        Properties props = new Properties();
+        props.put("mail.smtp.auth", "true");
+        props.put("mail.smtp.starttls.enable", "true");
+        props.put("mail.smtp.host", "smtp.gmail.com");
+        props.put("mail.smtp.port", "587");
+
+        Session session = Session.getInstance(props, new javax.mail.Authenticator() {
+            protected javax.mail.PasswordAuthentication getPasswordAuthentication() {
+                return new PasswordAuthentication("kinopol.rezerwacja@gmail.com", "qykZad-rojmyz-1howfe");
+            }
+        });
+        Message msg = new MimeMessage(session);
+        msg.setFrom(new InternetAddress("kinopol.rezerwacja@gmail.com", false));
+
+        msg.setRecipients(Message.RecipientType.TO, InternetAddress.parse("dejwid19996@wp.pl"));
```

Comment:
Email do kogo wysłać powinien być argumentem metody. 
Wiem, że nie jest to jeszcze skończone, ale już samo wysyłanie będzie gotowe. Tylko treść nie będzie.